materialdatabase/material_data_base_functions.py

```diff
@@ -153,6 +153,18 @@ def interpolate_a_b_c(a, b, c, no_interpolation_values=20):
     return a_reduced, b_interpol_common, c_interpol_common
 
 
+def interpolate_between_two_functions(x_data: list, y_data: list, x_new: list):
+    """
+    interpolates y_new on base of x_new with the (x_data,y_data)-dataset
```

Comment:
where are the two functions?

Reply:
one function is f(x_data) = y_data and the other (new) one is f(x_new) = y_new. the name of the function is irritating. i can change that. interpolate_between_two_graphs is probably a better name for the function.